app/views/layouts/application.html.erb

```diff
@@ -98,13 +98,13 @@
     <%= yield %>
   </main>
 
-  <footer class="usa-footer usa-footer-medium" role="contentinfo">
+  <footer class="usa-footer usa-footer-big" role="contentinfo">
```

Comment:
switch to big since medium has a small logo that doesn't balance well with our tall contact info on the right side.